pipelines/test/ldd-check.yaml

```diff
@@ -62,6 +62,19 @@ pipeline:
         fail "$f: missing ${missing# }"
```

Comment:
huh. i think the above line here should be 'return 0', not 'continue' as we're not in a loop here.